canonical/canonical.com
# Creation of OpenSearch Pages

## Done

- Updated /data/opensearch/
   - [PR](https://github.com/canonical/canonical.com/pull/1308)
- Created /data/opensearch/support
   - [PR](https://github.com/canonical/canonical.com/pull/1315)
- Created /data/opensearch/managed
   - [PR](https://github.com/canonical/canonical.com/pull/1317)
-  Created /data/opensearch/what-is-opensearch
   - [PR](https://github.com/canonical/canonical.com/pull/1311)
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Go through all the pages and check if all the pages are according to design.
- Check for all the links.
- Demo
   -  [home](https://canonical-com-1324.demos.haus/data/opensearch)
   -  [support](https://canonical-com-1324.demos.haus/data/opensearch/support)
   -  [managed](https://canonical-com-1324.demos.haus/data/opensearch/managed)
   -  [what-is-opensearch](https://canonical-com-1324.demos.haus/data/opensearch/what-is-opensearch)

## Issue / Card

Fixes #
- [WD-12223: Update /data/opensearch I Main Page](https://warthogs.atlassian.net/browse/WD-12223?atlOrigin=eyJpIjoiYmJkNTFhMWY1NjBiNDk2N2I1M2ViM2Q5NDJlZTQyZmUiLCJwIjoiaiJ9)
- [WD-12231: Create /data/OpenSearch/Support](https://warthogs.atlassian.net/browse/WD-12231?atlOrigin=eyJpIjoiMjQ5NzNlMjA4OWIyNDUzY2EzMDVmNDIzNzE2Y2QyOWMiLCJwIjoiaiJ9)
- [WD-12227: Create data/opensearch/what-is-opensearch](https://warthogs.atlassian.net/browse/WD-12227?atlOrigin=eyJpIjoiZDE5NDA0NWMwMjE3NGQ5YWJiZGU0YjNjMmEzNDJjNjAiLCJwIjoiaiJ9)
- [WD-12235: Create /data/OpenSearch/managed](https://warthogs.atlassian.net/browse/WD-12235?atlOrigin=eyJpIjoiMWNiNjc5MjExMWI1NGM0Y2I4NzJhMzE2YWM1NjJiODMiLCJwIjoiaiJ9)

## Screenshots

[if relevant, include a screenshot]
